docs/modules/ROOT/pages/servlet/authorization/method-security.adoc

```diff
@@ -51,6 +51,7 @@ Consider learning about the following use cases:
 * Coordinating with <<changing-the-order,@Transactional and other AOP-based annotations>>
 * Customizing <<customizing-expression-handling,SpEL expression handling>>
 * Integrating with <<custom-authorization-managers,custom authorization systems>>
+* Providing <<fallback-values-authorization-denied,fallback values when authorization is denied>>
```

Comment:
This is likely more valuable to readers than its position in the list. I think it likely belongs under "Authorizing methods with `@PreFilter` and `@PostFilter`". Up to you, though.